src/gsy_e/models/area/area.py

```diff
@@ -91,17 +91,15 @@ def __init__(self, name: str = None, children: List["Area"] = None,
     def get_state(self):
         """Get the current state of the area."""
         state = super().get_state()
-
-        state.update(**{
-            "current_tick": self.current_tick,
-            "area_stats": self.stats.get_state()
-        })
+        state.update({"current_tick": self.current_tick})
+        if not self.strategy:
```

Comment:
I assume that the strategy areas do not need the AreaStats and that's why this clause was added, right?

Reply:
This is my proposal actually. Currenlty, area_stats contain the followeing members:
```
exported_energy
imported_energy
rate_stats_market
```
which are market area specific. 
Do you foresee that the assets might also export statistics that should be part of the state in the future? 
If yes, let me revert this.

Reply:
This is only part of the state, therefore I do not think that it will be necessary for the future. Leave it as is and we will revert the clause if needed.